src/gltf-component-mappings.js

```diff
@@ -22,7 +22,10 @@ function registerRootSceneComponent(componentName) {
 registerRootSceneComponent("fog");
 registerRootSceneComponent("background");
 
-AFRAME.GLTFModelPlus.registerComponent("duck", "duck");
+AFRAME.GLTFModelPlus.registerComponent("duck", "duck", el => {
+  el.setAttribute("duck", "");
+  el.setAttribute("quack", { quackPercentage: 0.1 });
```

Comment:
Is this here just to avoid having to modify the gltf we have?

Reply:
makes it easier to change the quackPercentage rather than having to unbundle the glb, update the gltf, then rebundle it.